build.gradle

```diff
@@ -39,16 +39,59 @@ task replicateSources << {
 			def replicationProjects = replicationParent.listFiles().findAll(projectFilter)
 
 			def replicationProject = replicationProjects.find{ it.name == canonicalProject.name }
-			
+
 			def replicationSrc = new File( replicationProject, 'src/' )
-			
+
 			replicationSrc.deleteDir()
-			
+
 			def canonicalSrc = new File( canonicalProject, 'src/' )
-			
+
 			ant.copy(todir: replicationSrc) {
 				fileset(dir: canonicalSrc, includes: "**")
 			}
 		}
 	}
 }
+
+configurations {
+	bnd
+}
+
+dependencies {
+	bnd group: "biz.aQute.bnd", name: "biz.aQute.bnd", version: "3.1.0"
+}
+
+repositories {
+	mavenCentral()
+}
+
+task diff << {
+	[bndtoolsOutputFiles, gradleOutputFiles, mavenOutputFiles].each { replicationFiles ->
+		println "${replicationFiles}"
+		println "${replicationFiles}"
+		println "${replicationFiles}"
+		println "${replicationFiles}"
+
+		replicationFiles.each { replicationFile ->
+
+			println ""
+			println ""
+			println replicationFile.name
+			println "*******************************************************"
+
+			def liferayGradleOutputFile = liferayGradleOutputFiles.find {
+				it.name == replicationFile.name
+			}
+
+			String exclusions = "Archiver-Version, Javac-Debug, Javac-Deprecation, Javac-Encoding, Built-By, Build-Jdk, *pom.properties, *pom.xml, WEB-INF/lib/*"
+
+			javaexec {
+				args "diff", "-i", "${exclusions}", liferayGradleOutputFile, replicationFile
```

Comment:
Lets flip the order, replicataionFile should be first and liferayGradleOuputFile should be second, that way the output makes more sense.